plugins/tiddlywiki/menubar/menu.tid

```diff
@@ -7,34 +7,37 @@ tags: $:/tags/PageTemplate
 <$list filter="[<currentTiddler>addprefix[$:/config/plugins/menubar/MenuItems/Visibility/]get[text]] ~show +[match[show]]" variable="ignore">
 <$list filter="[[$size$]match[wide]] ~[<currentTiddler>get[show-when]match[$size$]] ~[{$:/state/popup/menubar/hamburger}match[yes]]" variable="ignore">
 <li style={{!!custom-menu-styles-$size$}} class={{{ [<currentTiddler>get[show-when]addprefix[tc-menubar-]] tc-menubar-item +[join[ ]] }}}>
-<$list filter="[<currentTiddler>!is-dropdown[yes]]" variable="listItem" emptyMessage="""
+<%if [<currentTiddler>!is-dropdown[yes]] %>
```

Comment:
I would prefer "positive" logic. So IMO we should use `is-dropdown[yes]` instead of `!is-dropdown`

Reply:
`[<currentTiddler>is-dropdown[yes]]` should be good, so imo compare is not needed